### DISCUSSION TITLE
auth middleware blocked bad JWT: parse token: token has invalid claims: token is expired

### DISCUSSION THREAD

**Discussion by bearded-papa** [garethgeorge/backrest#286] [Q&A] [answered]
I updated to v1.0.0 yesterday and all went well til I tried to log in this morning and it won't let me log in.
I use dockge and have seen the error message auth middleware blocked bad JWT: parse token: token has invalid claims: token is expired
How can I fix this

**Comment by garethgeorge** [comment, marked as answer]
Hey, go ahead and reset your password by editing your config and deleting the "users": ... section (typically near the bottom). You'll then be able to reconfigure your user / password.

**Reply by bearded-papa** [reply]
Thanks, 
I will keep that in mind if it happens again.
Luckily, I found I was still logged in on my mobile, so I just changed the password.
